packages/core/src/config.js

```diff
@@ -31,6 +31,7 @@ const DEFAULT_OPTIONS = {
     deprecated: "default",
   },
   skipDocDirective: [],
+  customDirective: {},
```

Comment:
I'd rather have the default undefined, see next comment
```suggestion
  customDirective: undefined,
```